app/views/shared/_profile_formfields.html.erb

```diff
@@ -84,7 +84,20 @@
       <%= f.text_area :bio_en, as: :text, class: 'mb input--width-100', rows: 10 %>
     </div>
 
-     <%= f.button :submit, t(:update, scope: 'profiles.form'), class:'btn btn--small mt mb++' %>
+    <%= f.button :submit, t(:update, scope: 'profiles.form'), class:'btn btn--small mt mb++' %>
```

Comment:
@tyranja dummerweise haben wir den hier in unserer Logik vergessen. Das Problem bleibt also, dass irgendwie die Profildaten gespeichert werden müssen, bevor man die andere Seite aufruft. Ich schaue mal ob man das über Javascript regeln kann.